assets/app/scripts/app.js

```diff
@@ -17,24 +18,121 @@ angular
     'ngSanitize',
     'ngTouch'
   ])
-  .config(function ($routeProvider, $locationProvider) {
+  // configure our tabs and routing
+  .config(['$routeProvider', 'HawtioNavBuilderProvider', function($routeProvider, builder) {
+    var template = function() {
+      // TODO - Don't love triggering the show/hide drawer here, would prefer if 
+      // we could listen for an event that the nav was being redrawn and 
+      // check HawtioNav.selected()
+      if (this.isSelected()) {
+        if (this.tabs && this.tabs.length > 0) {
+          $("body").addClass("show-drawer");
+        }
+        else {
+          $("body").removeClass("show-drawer");
+        }
+      }
+      return "<sidebar-nav-item></sidebar-nav-item>";        
+    };
+
+    var projectHref = function(path) {
+      return function() {
+        var injector = HawtioCore.injector;
```

Comment:
hmm, so HawtioCore is the global variable that hawtio is creating for itself and its plugins to use, specifically HawtioCore.injector is a shortcut for this:

HawtioCore.injector = angular.bootstrap(window.wrap(document), hawtioPluginLoader.getModules());